cdk_infra/lib/config/cluster-config/clusters.yml

```diff
@@ -45,6 +45,28 @@ clusters:
   - name: collector-ci-fargate-1-27
     version: "1.27"
     launch_type: fargate
+  - name: collector-ci-arm64-1-30
+    version: "1.30"
```

Comment:
updating [test_config](https://github.com/aws-observability/aws-otel-test-framework/blob/terraform/cdk_infra/test/test_config/test_clusters.yml) is probably the last change need